Dev

bartplessers opened the pull request:
See
https://github.com/Koenkk/zigbee2mqtt/issues/3237

small issues in documentation

Koenkk committed:
Promote to dev

Koenkk committed:
Update converters.

Koenkk committed:
Allow to specify adapter type. https://github.com/Koenkk/zigbee-herdsman/issues/72

Koenkk committed:
Update herdsman.

Koenkk committed:
Update converters.

Koenkk committed:
Make sure that there is always an update_available value for Home Assistant. https://github.com/Koenkk/zigbee2mqtt/issues/3146

Koenkk committed:
Fix unsupported device warnings when device is still being interviewed. #3158

Koenkk committed:
Log error when there is nothing to bind. #3159

Koenkk committed:
Update converters.

addisonlynch committed:
Add 75541 (#3167)

sjorge committed:
#3159 Fix incorrect unbind message (#3172)

Koenkk committed:
Update converters.

Nico87LBS committed:
Added Hue Econic Outdoor Pedestal (#3174)

Koenkk committed:
Update herdsman.

Koenkk committed:
Update herdsman and converters.

Koenkk committed:
Don't allow MQTT wildcard (# or +) in friendly_name. #3175

Nico87LBS committed:
Add Hue Impress outdoor Pedestal (#3186)

Koenkk committed:
Fix crash when removing device from group which has no devices in settings. #3185

Koenkk committed:
Fix typo.

Koenkk committed:
Don't allow non existing entities on availability_blacklist or availability_whitelist. #3191

Koenkk committed:
Update zigbee-herdsman. https://github.com/Koenkk/zigbee2mqtt/issues/3111

presslab-us committed:
Add humidity support for SmartThings (Centralite) 3310-S (#3197)

Koenkk committed:
Mock water_leak value for Home Assistant when it's not available. #3146

presslab-us committed:
Support wallyHOME Multi-Sensor (#3202)

presslab-us committed:
Support Centralite Night Light Repeater (#3203)

* Support Centralite Night Light Repeater

* Update homeassistant.js

Co-authored-by: Koen Kanters <koenkanters94@gmail.com>

Koenkk committed:
Update converters.

presslab-us committed:
Support Sylvania iQBR30 (#3205)

* Support Sylvania iQBR30

* Update homeassistant.js

Co-authored-by: Koen Kanters <koenkanters94@gmail.com>

Koenkk committed:
Update converters.

presslab-us committed:
Support Iris smart button 3460-L (#3214)

Koenkk committed:
Change unit of Home Assistant sensor_energy to kWh. https://github.com/Koenkk/zigbee-herdsman-converters/issues/974

Koenkk committed:
Cover position for ZM-CSW032-D. https://github.com/Koenkk/zigbee2mqtt/issues/3216

Koenkk committed:
Allow to change non-setted options via device_options. #3215

Koenkk committed:
Update herdsman.

Koenkk committed:
Remove battery_low for HS1CG-E. https://github.com/Koenkk/zigbee2mqtt/issues/3041

presslab-us committed:
Support Iris hose faucet water timer 27087-03 (#3230)

presslab-us committed:
Fix power reporting of Centralite 4257050-RZHAC (#3236)

Co-authored-by: Koen Kanters <koenkanters94@gmail.com>

presslab-us committed:
Support Centralite thermostat 3157100 (#3234)

Co-authored-by: Koen Kanters <koenkanters94@gmail.com>

Koenkk committed:
Update converters.

presslab-us committed:
Fix colortemp ability for Paulmann RGBW Light (#3239)

Koenkk commented:
I don't understand this PR, regarding #3237 I would expect a PR to https://github.com/koenkk/zigbee2mqtt.io

danieledwardgeorgehitchcock committed:
Added device support for Hue Beyond suspension light (#3246)

Koenkk committed:
DeviceReport: don't crash on binds to non-existing entities. #3245

Koenkk committed:
Update herdsman and converters.

Koenkk committed:
Fix tests.

presslab-us committed:
Support Leviton Omnistat2 thermostat (#3249)

* Support Leviton Omnistat2 thermostat

* Update homeassistant.js

Co-authored-by: Koen Kanters <koenkanters94@gmail.com>

presslab-us committed:
Support Orvibo RL804QZB (#3252)

Co-authored-by: Koen Kanters <koenkanters94@gmail.com>

Koenkk committed:
Update converters.

Koenkk committed:
Update herdsman.

Koenkk committed:
Only setup supported color capabilities. https://github.com/Koenkk/zigbee2mqtt/issues/3260

presslab-us committed:
Fix definition of Nyce NCZ-3011-HA (#3267)

Koenkk committed:
Update herdsman and converters.

Koenkk committed:
Update herdsman

dzegarra committed:
Added HA config for radiator valve Siterwell GS361 (#3259)

* Added HA config for radiator valve Siterwell GS361

* Updated payload and state keywords for mqtt msgs

* Update homeassistant.js

* Update homeassistant.js

* Update homeassistant.js

Co-authored-by: Koen Kanters <koenkanters94@gmail.com>

Koenkk committed:
ZM-CSW032-D doesn't support reporting. https://github.com/Koenkk/zigbee2mqtt/issues/3216

Koenkk committed:
Update converters.

kiall committed:
Fix conflict between illuminance and illuminance_lux (#3275)

* Fix conflict between illuminance and illuminance_lux

On certain devices, e.g. GZCGQ01LM, 9290012607, 9290019758, TERNCY-PP01 etc, both
cfg.sensor_illuminance and cfg.sensor_illuminance_lux are applied. As both of these
had the same object_id, only one would show in Home Assistant. Oddly, a mix of both
would apply - e.g. I would get the illuminance value, with the illuminance_lux unit
of 'lx'.

* Add a test for duplicated type/object_id in HA discovery configs

Koenkk committed:
Update converters.

Koenkk committed:
Update converters.

sjorge committed:
entityPublish now passes in meta.mapped as an array for groups (#3283)

Koenkk committed:
Add missing devices to configuration. #3263

Koenkk committed:
Allow to disable log rotation. #3280

Koenkk committed:
Start on legacy api isolation. https://github.com/Koenkk/zigbee2mqtt/issues/3281

Koenkk committed:
Refactor

Koenkk committed:
Refactor

Koenkk committed:
Don't cache properties which start with action_

Koenkk committed:
Move stateChange event to event bus.

Koenkk committed:
Isolate legacy MQTT log. #3281

Koenkk committed:
Refactor

Koenkk committed:
Update herdsman.

averbeck committed:
Action and click trigger for multiple endpoints and 511.344 / ZG2819S support (#3289)

* Home Assistant support for ZG2819S/511.344

* Added 'homeassistant_legacy_triggers' support for devices with multiple endpoints

* Changed code style to pass lint test

* Reverted fix for the action and click trigger to send an empty message for devices with multiple endpoints

* Home Assistant support for the updated ZG2819S converter

* Update homeassistant.js

Co-authored-by: Koen Kanters <koenkanters94@gmail.com>

Koenkk committed:
Update herdsman.

Koenkk committed:
Update converters.
